heron/metricsmgr/src/java/com/twitter/heron/metricsmgr/Metrics.java

```diff
@@ -0,0 +1,41 @@
+//  Copyright 2017 Twitter. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+package com.twitter.heron.metricsmgr;
+
+import com.twitter.heron.spi.metricsmgr.metrics.MetricsRecord;
+
+public final class Metrics {
```

Comment:
This class seems like a Util class. The class name may be Util rather than Metrics

Reply:
I will update the name.